webpack/ForemanRhCloudFills.js

```diff
@@ -15,7 +15,7 @@ const fills = [
   },
   {
     slot: 'host-details-page-tabs',
-    name: 'Insights',
+    name: __('Insights'),
```

Comment:
Please don't change the `name` property. You can add a `title` under `metadata`: https://github.com/theforeman/foreman/blob/770e8cc3527a45a76358bf529f17dab63145fdf9/webpack/assets/javascripts/react_app/components/HostDetails/index.js#L267
The `name` is used as a key.